gauntlet/packages/gauntlet-solana-contracts/artifacts/words/words.txt

```diff
@@ -0,0 +1,2048 @@
+abandon
```

Comment:
Let's be more explicit about what these words are. Consider renaming the file as `bip-0039/english.txt`.